idd/Energy+.idd.in

```diff
@@ -8701,6 +8701,15 @@ WindowThermalModel:Params,
        \units Pa
        \minimum> 0
        \default 101325
+       
+WindowsCalculationEngine,
+       \memo This object is used to switch algorithms between built in windows engine 
+       \memo and windows engine as exterior component
```

Comment:
This sentence is pretty vague, if any other last minute code changes are needed, I'd like this to be explained a little better.

Reply:
You are right. This can be better explained. Will do.